src/PyNvCodec/__init__.pyi

```diff
@@ -7,8 +7,6 @@ BGR: PixelFormat
 BIT_DEPTH_MINUS_8: NV_DEC_CAPS
 BT_601: ColorSpace
 BT_709: ColorSpace
-BY_NUMBER: SeekCriteria
```

Comment:
I feel like a better wording would be `BY_FRAME`.

Reply:
But `SeekCriteria` is removed in this patch, or am I missing something?

Reply:
My bad - looked at the wrong side ...